### DISCUSSION TITLE
Hide events in agenda widget?

### DISCUSSION THREAD

**Discussion by nofishonfriday** [FossifyOrg/General-Discussion#451] [Q&A] [answered]
Hi,
can I somehow hide an event in the agenda widget?
Like a birthday date. No need (for me) to always see it during the year (just clutters the view), as I've set a reminder for it anyway.

**Comment by Aga-C** [comment, marked as answer]
Currently, it's not possible, but there's a feature request for this: https://github.com/FossifyOrg/Calendar/issues/4.

**Reply by nofishonfriday** [reply]
Thanks for the prompt reply. :)
